app/controllers/agenda-items/index.ts

```diff
@@ -171,7 +172,10 @@ export default class AgendaItemsIndexController extends Controller {
   @tracked plannedStartMax = '';
   @tracked governingBodyClassifications = '';
 
-  @tracked showAdvancedFilters = this.governingBodyClassifications.length > 0;
+  @computed('governingBodyClassifications.length')
```

Comment:
**Blocker:** Using a getter is enough, `@computed` decorator is not needed. 
In the previous version, `this.governingBodyClassifications.length > 0` was only evaluate and pass to `showAdvancedFilters` only once at initialization. With the getter, the component will know and refresh each time `governingBodyClassifications` change. cf. https://guides.emberjs.com/v4.12.0/in-depth-topics/autotracking-in-depth/